buildscripts/Castle.Core.nuspec

```diff
@@ -14,26 +14,30 @@
     <copyright>Copyright 2004-$CurrentYear$ Castle Project - http://www.castleproject.org/</copyright>
     <tags>castle dynamicproxy dynamic proxy dynamicproxy2 dictionaryadapter emailsender</tags>
     <dependencies>
-      <group targetFramework=".NETPlatform5.4">
-        <dependency id="System.AppContext" version="4.0.0" />
-        <dependency id="System.Collections.Specialized" version="4.0.0" />
-        <dependency id="System.ComponentModel.TypeConverter" version="4.0.0" />
-        <dependency id="System.Console" version="4.0.0-beta-23516" />
-        <dependency id="System.Diagnostics.Tools" version="4.0.0" />
-        <dependency id="System.Diagnostics.TraceSource" version="4.0.0-beta-23516" />
-        <dependency id="System.Dynamic.Runtime" version="4.0.10" />
-        <dependency id="System.Globalization" version="4.0.10" />
-        <dependency id="System.Linq" version="4.0.0" />
-        <dependency id="System.ObjectModel" version="4.0.10" />
-        <dependency id="System.Reflection" version="4.0.10" />
-        <dependency id="System.Reflection.Emit" version="4.0.0" />
-        <dependency id="System.Reflection.Emit.Lightweight" version="4.0.0" />
-        <dependency id="System.Reflection.Extensions" version="4.0.0" />
-        <dependency id="System.Reflection.TypeExtensions" version="4.0.0" />
-        <dependency id="System.Runtime.Extensions" version="4.0.10" />
-        <dependency id="System.Threading" version="4.0.10" />
-        <dependency id="System.Xml.XmlDocument" version="4.0.0" />
-        <dependency id="System.Xml.XmlSerializer" version="4.0.10" />
+      <group targetFramework=".NETStandard1.3">
+        <dependency id="System.AppContext" version="[4.1.0-rc2-24027, )" />

```

Comment:
Specifying these as `<dependency id="System.AppContext" version="4.1.0-rc2-24027" />` is a more idiomatic way of expressing this version constraint. I know `dotnet build` produces versions in this format, but it should eventually go away. (https://github.com/dotnet/cli/issues/2893)